web_src/less/_repository.less

```diff
@@ -3237,3 +3237,31 @@ td.blob-excerpt {
   transform: scale(105%);
   box-shadow: 0 .5rem 1rem var(--color-shadow) !important;
 }
+
+@media @mediaSm {
+  .repository.file.list {
+    #repo-files-table {
+      .entry,
+      .commit-list {
+        display: flex !important;
+        padding-top: 4px;
+        padding-bottom: 4px;
+
+        td.age,
+        th.age {
+          margin-left: auto;
+        }
+
+        td.message,
+        span.commit-summary {
+          display: none !important;
+        }
+
+        th .mobile-align {
+          display: inline-block;
+          padding-top: 5px;
```

Comment:
This and the paddings above, do you mean something `align-items: center;` with flexbox?

And the <span> seems not necessary, a CSS named can be added to the `th` IMO.

Reply:
https://github.com/go-gitea/gitea/pull/19515/commits/21ceac153912026593f2120e5ab8ea514935fb8b